tough/src/editor/signed.rs

```diff
@@ -102,18 +105,22 @@ where
         Ok(signed_role)
     }
 
+    /// Provides access to the internal signed metadata object.
     pub fn signed(&self) -> &Signed<T> {
         &self.signed
     }
 
+    /// Provides access to the internal buffer.
```

Comment:
```suggestion
    /// Provides access to the internal buffer containing the serialized form of the signed role.
```

Reply:
I might add the text:

`This buffer should be used anywhere this role is written to file` or something to that effect.